wled00/espnow_broadcast.cpp

```diff
@@ -385,18 +397,18 @@ void ESPNOWBroadcastImpl::onESPNowRxCallback(const uint8_t *mac, const uint8_t *
         // be safe about accessing memory that isn't directly exposed to the callback
         rssi = 0;
     }
-    // Serial.printf( "RX from " ); 
-    // logMACAddr(mac);
-    // Serial.printf( " %d:bytes rssi:%d\n", len, rssi);
 
     if (!espnowBroadcastImpl.queuedNetworkRingBuffer.push(mac, data, len, rssi)) {
-        if (len > sizeof(WLED_ESPNOW_MAX_MESSAGE_LENGTH)) {
-#ifdef ESPNOW_DEBUGGING
-            Serial.printf("Receive to large of packet %d bytes. ignoring...\n", len);
+        Serial.printf("Failed to queue message (%d bytes) to ring buffer.  Dropping message\n", len);
+    } else {
+#ifdef ESPNOW_CALLBACK_DEBUGGING
```

Comment:
log this via a separate macro since it happens from a different thread and can break up other serial messages in node.h